ddl/backfilling.go

```diff
@@ -492,14 +514,15 @@ func (dc *ddlCtx) handleRangeTasks(sessPool *sessionPool, t table.Table, workers
 		}
 
 		task := &reorgBackfillTask{
+			id:              i,
 			physicalTableID: physicalTableID,
 			startKey:        keyRange.StartKey,
 			endKey:          endKey,
 			// If the boundaries overlap, we should ignore the preceding endKey.
 			endInclude: endK.Cmp(keyRange.EndKey) != 0 || i == len(kvRanges)-1}
 		batchTasks = append(batchTasks, task)
 
-		if len(batchTasks) >= len(workers) {
+		if len(batchTasks) >= batchSize {
```

Comment:
Why here use batchSize to set batchTasks?

Reply:
It is fixed to be `1024` to limit the memory usage.